docs/guide/yaml/test/index.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Testing YAML Blueprints
+layout: website-normal
+children:
+- test-entities.md
+- usage-examples.md
+---
+
+Brooklyn provides a selection of basic test entities which can be used to validate Blueprints via YAML. These are divided into two groups; structural, which effect the order in which child entities are started; and validation, which are used to confirm the application is deployed as intended.
```

Comment:
Instead of "order in which child entities are started", perhaps relate it to the order in which child test-things are run. But obviously not with that wording! I'm fine with leaving it as-is if we can't come up with a better term for "child test-things".